src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  transition: 300ms ease;
+
+  &:hover {
+    transform: scale(1.2);
```

Comment:
The scale transformation on hover is quite large (1.2). Ensure this is the intended design, as it might cause layout issues or overlap with other elements.